nemo/collections/tts/g2p/models/zh_cn_pinyin.py

```diff
@@ -201,7 +202,10 @@ def __call__(self, text: str) -> List[str]:
             tone_hyp = pinyin[-1]
             if tone_hyp in self.tone_dict:
                 syllable = pinyin[:-1]
-                assert syllable in self.phoneme_dict, f"Syllable <{syllable}> does not exist in the dictionary."
+                if syllable not in self.phoneme_dict:
+                    err = True
+                    logging.error(f"Syllable <{syllable}> does not exist in the dictionary.")
```

Comment:
This change may lead to deletion errors in the synthesized speech. it would be better to add a TODO note showing potential symbol deletion risks and will fix it in future.